cookbook-react-native/recipes/pickers/dateTimePicker/interfaces.ts

```diff
@@ -0,0 +1,36 @@
+import {
+  ImageSourcePropType,
+  ImageStyle,
+  TextStyle,
+  ViewStyle
+} from 'react-native';
+
+export interface IDateTimePicker {
+  accessibilityLabel: string;
+  styleModal?: ViewStyle;
+  styleContainer?: ViewStyle;
+  mode?: 'date' | 'time' | 'datetime';
+  is24hs: boolean;
+  setDate?: Function;
+  setHour?: Function;
+  styleBtn?: ViewStyle;
+  styleTextBtn?: TextStyle;
+  textButton?: string;
+  onPressButton?: Function;
+  onPressConfirm?: Function;
+  onPressCancel?: Function;
+  icon?: ImageSourcePropType;
+  styleIcon?: ImageStyle;
+  hourFormat?: 'toLocaleTimeString' | 'toISOString';
+  confirmTextIOS?: string;
+  cancelTextIOS?: string;
+}
+
+export interface ITouchButton {
+  styleBtn?: ViewStyle;
+  showDatePicker?: () => void;
```

Comment:
change `Function` typing above like this or receiving corresponding params

Reply:
changed